mlir/utils/performance/analysis/analizeDataAvarege.py

```diff
@@ -0,0 +1,68 @@
+import sys
+import argparse
+import glob
+import pandas as pd
+
+def analizeData(output):
+    # Data on the paths conv/tunedData, gemm/tunedData are obtained as output files
```

Comment:
Sorry about being pedantic, but can you write the exact command you used? My point is that people like @ethansaurusrex should be able to immediately reproduce your results by reading this script

Reply:
There is a usage comment right below the main function

Reply:
IIUC that is the usage of the script. I am talking about the command to generate the `.debug` files.